modules/ITK/cli/statismo-build-gp-model.cxx

```diff
@@ -136,7 +136,16 @@ bool isOptionsConflictPresent(programOptions& opt) {
         return true;
     }
 
-    if (opt.strOutputFileName == "" || opt.strReferenceFile == "") {
+    if (opt.strOptionalModelPath == "" && opt.strReferenceFile == "") {
+        return true;
+    }
+
+    if (opt.strOptionalModelPath != "" & opt.strReferenceFile != "") {
+        return true;
+    }
```

Comment:
Exactly one has to be set could be represented with xor:
if (((opt.strOptionalModelPath == "") ^ (opt.strReferenceFile == "")) == false){return true;}